client/files.go

```diff
@@ -15,82 +15,106 @@
 package client
 
 import (
-	"errors"
 	"io/fs"
```

Comment:
Oh, this is interesting -- now that I think of it, `io/fs` was added in Go 1.16, and we currently support Go versions down to 1.14. So we should probably make this `os.FileInfo` after all.